client/src/hooks/useHttp.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+
+const baseUrl = 'http://localhost:4000/api/';
+
+const baseAxios = axios.create({
+  baseURL: baseUrl
```

Comment:
If name of key and properties is equal u may write it as one name

Reply:
https://github.com/airbnb/javascript#es6-object-concise